src/components/duende-og-image.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
```

Comment:
Noooooooooooooooo! It's fine.

Reply:
It's only for image generation, don't worry. It isn't going to make it past the build process. EVER!